src/bin/sol-fbp-runner/Widget.js

```diff
@@ -0,0 +1,222 @@
+(function(window) {
+
+	'use strict'
+
+	function Widget(payload, key) {
+
+		//caching a secure reference to the instance
+		var instance = this;
+		//var selectedPort;
+
+		Widget.prototype.portsCount = 0;
+
+		//unique id for the widget and other unique properties
+		this.raw = JSON.stringify(payload, null, 4);
```

Comment:
keeping this is not that good, is this a stop-gap until we can render this in a nice format/html?